_pages/meetings.md

```diff
@@ -14,8 +14,12 @@ Meetings and telecon times are available as a [Google calendar](https://calendar
 
 ### Community Meetings
 
+The 2020 Fall Python in Heliophysics Community Meeting **will be held remotely** on Zoom (see details on the meeting's web page - coming soon), spanning over the course of four weeks. The meeting will begin on Monday, October 26th, 2020, and end Monday, November 16th, 2020.
```

Comment:
No comma is needed after `Monday, October 26th, 2020` because that isn't a compound sentence. 😬

Reply:
Okie doke to this and below grammar comments.